src/tests/test_telemetry.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+from unittest.mock import ANY, Mock
+import pytest
+import urllib.request
+import duckdb
+from sql.telemetry import telemetry
+from sql import plot
+
+
+@pytest.fixture
+def simple_db_conn():
+    if not Path("iris.csv").is_file():
+        urllib.request.urlretrieve(
+            "https://raw.githubusercontent.com/plotly/datasets/master/iris-data.csv",
+            "iris.csv",
+        )
+    conn = duckdb.connect(database=":memory:")
+    return conn
+
+
+@pytest.fixture
+def mock_log_api(monkeypatch):
```

Comment:
I don't think we need this as it's inherited from ploomber-core.

Reply:
Mind explaining more or hinting me some examples?
My question is how to test whether the log_api was called or not if we don't mock the log_api

Reply:
I thought this fixture already exists in the core module at confest, but I don't see it, we can resolve this.